<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogHotkey</class>
 <widget class="QDialog" name="DialogHotkey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotkey</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QtExtKeySequenceEdit" name="show_groups"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Show routes</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QtExtKeySequenceEdit" name="show_routes"/>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QtExtKeySequenceEdit" name="show_mainwindow"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Show groups</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Trigger main window</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>System Proxy</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QtExtKeySequenceEdit" name="system_proxy"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExtKeySequenceEdit</class>
   <extends>QKeySequenceEdit</extends>
   <header>3rdparty/QtExtKeySequenceEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>show_mainwindow</tabstop>
  <tabstop>show_groups</tabstop>
  <tabstop>show_routes</tabstop>
  <tabstop>system_proxy</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogHotkey</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogHotkey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
